src/api/json/catalog.json

```diff
@@ -411,6 +411,24 @@
         "1.3": "https://json.schemastore.org/azure-iot-edge-deployment-template-4.0.json"
       }
     },
+    {
+      "name": "buf.yaml",
+      "description": "https://buf.build/docs/configuration/v1/buf-yaml",
```

Comment:
not sure if it's preferred that these names/descriptions are more plaintext and descriptive, but I don't have a suggestion off-hand.

Reply:
I think there might be a few other entries that use URLs as their descriptions, but if you can use plaintext, that would probably be better. I believe `description` shows up in some plain-text GUI contexts that prevent them from being clickable.

Reply:
cool - I changed these to the descriptions from the schemas themselves in https://github.com/SchemaStore/schemastore/pull/3310/commits/d6b605801aa025ff394ac9d5b6fd38884f46deec.

Reply:
(And then removed the ending periods due to the lint check in https://github.com/SchemaStore/schemastore/pull/3310/commits/116a415dcb0bd86f76b9139c1926a7d9f521c2f6.)